source/process/release-process.md

```diff
@@ -224,6 +224,7 @@ The final release is cut. If an urgent and important issue needs to be addressed
     - Review any changes made to install guides, and test if necessary
     - Run loadtests against the final release build to confirm there are no performance issues
     - Review [Security Policies](https://docs.mattermost.com/process/security.html) page is updated
```

Comment:
This wasn't part of AB's updates but wondering whether this should read:
1. Review changes made to [Security Policies](https://docs.mattermost.com/process/security.html) page
or
2. Ensure [Security Policies](https://docs.mattermost.com/process/security.html) page has been updated

Reply:
@lindy65 I'd go with two, agree